android/app/src/main/java/org/fossasia/openevent/adapters/BaseRVAdapter.java

```diff
@@ -59,9 +59,12 @@ public void clear() {
         notifyDataSetChanged();
     }
 
+    /**
+     * Return size of dataList and 0 if it is null.
+     */
     @Override
     public int getItemCount() {
-        return dataList.size();
+        return dataList == null ? 0 : dataList.size();
```

Comment:
This will introduce subtle bugs, the app should crash if null list is passed

Reply:
Oh ok i will revert that.